backend/tests/services/test_knowledge_qa_history.py

```
@@ -0,0 +1,425 @@
+            start_time=start_time,
+            end_time=end_time,
+        )
+
+        assert isinstance(result, QAHistoryResponse)
+        assert result.items == []
+        assert result.pagination.total == 0
+        assert result.pagination.page == 1
+        assert result.pagination.page_size == 20
+        assert result.pagination.total_pages == 1
+
+    def test_get_qa_history_with_pagination(self, test_db: Session, test_user):
```

Ruff (0.14.10)

Unused method argument: `test_user`

(ARG002)